frontend/src/components/Pages/MissionDefinitionPage/MissionDefinitionPage.tsx

```diff
@@ -76,6 +97,32 @@ const MissionDefinitionPageBody = ({ missionDefinition }: { missionDefinition: M
         }
     }
 
+    const onDeleteAutoSchedule = () => {
+        const defaultMissionDefinitionForm: MissionDefinitionUpdateForm = {
+            comment: missionDefinition.comment,
+            inspectionFrequency: missionDefinition.inspectionFrequency,
+            autoScheduleFrequency: undefined,
+            name: missionDefinition.name,
+            isDeprecated: false,
+        }
```

Comment:
```suggestion
        const defaultMissionDefinitionForm: MissionDefinitionUpdateForm = {
            autoScheduleFrequency: undefined,
        }
```
I think this should work because it does on the swagger endpoint at least

Reply:
<img width="1268" alt="image" src="https://github.com/user-attachments/assets/fbe20f0f-c8ff-4320-9a38-557e8d0977f5" />
Cannot remove name, but can remove the rest

Reply:
Nvm, if we dont set the rest they become undefined. The result will be that removing autoSchedule also removes inspectionFrequency and comment

Reply:
This should be possible but it's kinda of scope creep so we can take it as improvement for the future